pyiron_atomistics/lammps/interactive.py

```diff
@@ -108,30 +108,20 @@ def interactive_cells_setter(self, cell):
                 "Warning: setting upper trangular matrix might slow down the calculation"
             )
 
-        is_skewed = self._prism.is_skewed()
-        is_scaled = self.structure._is_scaled
-        if is_scaled:
-            warnings.warn('set_relative() is deprecated as of 2020-02-26. It is not guaranteed from pyiron_atomistics vers. 0.3')
+        is_skewed = self._structure_current.is_skewed()
+        was_skewed = self._structure_previous.is_skewed()
 
-        if is_skewed and is_scaled:
-            self._interactive_lib_command(
-                "change_box all triclinic"
-            )
+        if is_skewed:
+            if not was_skewed:
+                self._interactive_lib_command(
+                    "change_box all triclinic"
+                )
             self._interactive_lib_command(
                 "change_box all x final 0 %f y final 0 %f z final 0 %f \
                  xy final %f xz final %f yz final %f remap units box"
                 % (lx, ly, lz, xy, xz, yz)
             )
-        elif is_skewed and not is_scaled:
-            self._interactive_lib_command(
-                "change_box all triclinic"
-            )
-            self._interactive_lib_command(
-                "change_box all x final 0 %f y final 0 %f z final 0 %f \
-                xy final %f xz final %f yz final %f units box"
-                % (lx, ly, lz, xy, xz, yz)
-            )
-        elif not is_skewed and is_scaled:
+        elif was_skewed:
             self._interactive_lib_command(
```

Comment:
I know that this line is not part of the PR, but why is it setting first triclinic and then reverting to orhonormal?

Reply:
Good point. I changed it.

It was because in the old implementation we didn't check beforehand whether the box was skewed or not. With this, we make sure that the box first becomes triclinic, which allows us to change the shear components to 0.